akka-osgi/build.sbt

```diff
@@ -1,7 +1,7 @@
-import akka.{ AkkaBuild, Dependencies, Formatting, OSGi }
+import akka.{ AkkaBuild, Dependencies, Formatting, OSGi, Dist }
 
 AkkaBuild.defaultSettings
-AkkaBuild.dontPublishSettings
+Dist.includeInDist := false
```

Comment:
Could it be that it duplicates stuff? I'd rather make it behave the same and have someone convince me to change it than to change it without knowing though :)